apps/src/templates/InlineMarkdown.jsx

```diff
@@ -27,7 +27,7 @@ export default class InlineMarkdown extends React.Component {
   };
 
   render() {
-    const rendered = markdownToReact.processSync(this.props.markdown).contents;
+    const rendered = markdownToReact.processSync(this.props.markdown).result;
```

Comment:
note about this change: the [unified](https://unifiedjs.com/) library was upgraded with storybook, and the structure of the object returned from `processSync` changed with that upgrade. there are unit/UI tests that cover this and i manually tested components to make sure nothing changed